python/opendataloader-pdf-mcp/src/opendataloader_pdf_mcp/jobs.py

```
@@ -0,0 +1,192 @@
+
+def _now() -> str:
+    return datetime.now(timezone.utc).isoformat()
+
+
+class JobManager:
+    def __init__(self) -> None:
+        self._jobs: dict[str, Job] = {}
+        self._hash_index: dict[str, str] = {}  # content_hash → job_id (DONE only)
+        self._lock = threading.Lock()
+
+    def submit(self, path: str, format: str, **kwargs: Any) -> str:
```

Ruff (0.15.12)

[error] Function argument `format` is shadowing a Python builtin

(A002)